### ISSUE TITLE
Is it possible to query the rows inside the dataframes directly?

### ISSUE THREAD

**Issue by rdaher** [dpguthrie/yahooquery#15]
Awesome job on the project! thank you so much!!!! 

If I wanted to quickly extract a piece of info, I found myself getting the database 
(example:  aapl.balance_sheet)
Next I have to list the rows it has, and identify the one I need, and then extract that.
Then if I wanted to extract an item from the financial statement I had to go to aapl.income_statement(), the load the rows, identify the one I want and call its data

Is there a possibility to query the rows directly after importing yahooquery even if they'd belong to different dataframes?

ex:
from yahooquery import Ticker 
import pandas as pd
aapl = Ticker('aapl') 
aapl.cashAndCashEquivalent #would immediately pull the cash and cash equivalent from the balance sheet data
aapl.TotalDebt #would immediately pull apple's total debt from the balance sheet data
aapl.TotalOperatingIncomeAsReported would pull that data from the income statement database.
aapl.enterpriseValue would immediately pull that number from the ticker.key_stats data...

Thank you so much again! appreciate you sharing!!!!

**Comment by dpguthrie** [comment]
I think that's an interesting use case.  I could probably accommodate most of it, and here's why / how:

The balance sheet, cash flow, income statement, and valuation measures data are retrieved from the same endpoint just with different query parameters.  I could create an additional method that allows the user to specify what data from each they would like to retrieve.  Something like:

```
from yahooquery import Ticker

aapl = Ticker('aapl')
data = aapl.get_financial_data(['CashAndCashEquivalents', 'TotalDebt', 'TotalOperatingIncomeAsReported', 'EnterpriseValue'], frequency='q')
```

The slight issue would then be incorporating data from other properties / methods like key_stats.  You could probably do it but it's just not as clean as you would have to make more than one request depending on what data you're trying to retrieve.  

I'll most likely include the feature above though with the next release.

**Comment by rdaher** [comment]
Thank you!!

**Comment by dpguthrie** [comment]
Version 2.2.5 has this feature.  Use the code below:

```
from yahooquery import Ticker
aapl = Ticker('aapl')

data = aapl.get_financial_data(['CashAndCashEquivalents', 'TotalDebt', 'TotalOperatingIncomeAsReported', 'EnterpriseValue'], frequency='q')
```

That will also return the TTM data, which, I believe, you didn't want.  You can get rid of that with the following:

```
data = aapl.get_financial_data(['CashAndCashEquivalents', 'TotalDebt', 'TotalOperatingIncomeAsReported', 'EnterpriseValue'], frequency='q', trailing=False)
```

**Comment by rdaher** [comment]
Thank you much!! 
I will give a try soon!

**Comment by rdaher** [comment]
Thank you Doug!
Had a chance to test the changes lately. All good. One thing that is not working for me anymore is the ticker.news
I tried it with different stocks and it would not work.
Input:

symbol = 'MYOV'
ticker = Ticker(symbol)
ticker.news

Output:

<bound method Ticker.news of <yahooquery.ticker.Ticker object at 0x7fb36407d390>>

Best
Ralph

On Aug 3, 2020, at 12:36 AM, Doug <notifications@github.com> wrote:

﻿

Closed #15<https://github.com/dpguthrie/yahooquery/issues/15>.

**Comment by dpguthrie** [comment]
It is now a method that takes arguments.

https://dpguthrie.github.io/yahooquery/guide/ticker/miscellaenous/#news
